mule-packager/src/main/java/org/mule/tools/api/packager/archiver/MuleArchiver.java

```diff
@@ -57,4 +62,14 @@ public void addWsdl(File resource, String[] includes, String[] excludes) throws
       addClasses(resource, includes, excludes);
     }
   }
+
+  /**
+   * @param resource Folder which contents are going to be added to the lib folder.
+   * @return
+   */
+  public void addLib(File resource, String[] includes, String[] excludes) {
```

Comment:
I reckon the test I created in 3.x for this is not enough 
Could we please create a jira to update the test of this class and validate it properly ?

Reply:
Ok!